Falco build failure

```
2024/09/16 08:11:50 WARN CMake Error at build/falcosecurity-libs-repo/falcosecurity-libs-prefix/src/falcosecurity-libs/cmake/modules/protobuf.cmake:26 (message):
2024/09/16 08:11:50 WARN   Couldn't find system protobuf
```

--- a/grpc.yaml
+++ b/grpc.yaml
@@ -1,7 +1,7 @@
 package:
   name: grpc
   version: 1.65.5
-  epoch: 0
+  epoch: 1
   description: The C based gRPC
   copyright:
     - license: Apache-2.0 AND BSD-3-Clause AND MIT

